docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -5,4 +5,254 @@ navigation:
+
+#### In Next.js
+
+```tsx
+// /apps/stores/store-b/storefront-unified-nextjs/components/example-component.tsx
+
+interface ExampleComponentProps {
+  title: string;
+  imageSrc: string;
+  customSubtitle?: string; // New optional prop
+}
+
+export default function ExampleComponent({ title, imageSrc, customSubtitle }: ExampleComponentProps) {
+  return (
+    <div className="store-b example-component">
+      <img src={imageSrc} alt={title} />
+      <h2>{title}</h2>
+      {customSubtitle && <p className="subtitle">{customSubtitle}</p>} {/* Customized subtitle */}
+    </div>
+  );
+}
+```
+
+#### In Nuxt
+
+```vue
+// /apps/stores/store-b/storefront-unified-nuxt/components/ExampleComponent/ExampleComponent.vue
+
+<template>
+  <div class="store-b example-component">
+    <img :src="imageSrc" :alt="title" />
+    <h2>{{ title }}</h2>
+    <!-- Customized subtitle for store-b -->
+    <p v-if="customSubtitle" class="subtitle">{{ customSubtitle }}</p>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+   title: string;
+   imageSrc: string;
+   customSubtitle?: string; // New optional prop
+}>();
+</script>
+```
+
+#### Scenario 2: Removing Props
+
+In this scenario, we'll demonstrate removing the `imageSrc` prop because the component no longer displays an image.
+
+1. **Identify Components That Need Changes**: Determine which components need the prop removed.
+2. **Create a Plan for Migration**: Decide how to handle existing usages of the removed prop.
+3. **Update Component and Its Usages**: Remove the prop and update all component instances.
+
+#### In Next.js
+
+```tsx
+// /apps/stores/store-a/storefront-unified-nextjs/components/example-component.tsx
+
+interface ExampleComponentProps {
+  title: string;
+  // imageSrc prop removed
+}
+
+export default function ExampleComponent({ title }: ExampleComponentProps) {
+  return (
+    <div>
+      {/* Image element removed */}
+      <h2>{title}</h2>
+    </div>
+  );
+}
+```
+
+#### In Nuxt
+
+```vue
+// /apps/store-a/storefront-unified-nuxt/components/ExampleComponent/ExampleComponent.vue
+
+<template>
+  <div>
+    <!-- Image element removed -->
+    <h2>{{ title }}</h2>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  title: string;
+  // imageSrc prop removed
+}>();
+</script>
+```
+
+### Handling the Migration
+
+When removing props like `imageSrc`, consider these strategies:
+
+1. **Communicate Changes**: Notify all teams about the breaking change.
+2. **Use TypeScript**: Let TypeScript help identify places where the removed prop is still being used.
+3. **Implement Gradually**: For large codebases, consider making the prop optional first before removing it completely.
+4. **Documentation**: Update component documentation to reflect the new props interface.
+
+This approach ensures a smooth transition when removing props from components while minimizing disruption across your multibrand project.
```

Comment:
This is very vague. Eg. I don't know the consequences of removing `imageSrc` that can happen, so I don't know how would I communicate it to the team.

Reply:
I've removed it and added info for removing props safely in the Removing Props part.